outbox/docker-compose.yaml

```diff
@@ -65,50 +63,31 @@ services:
      - KAFKA_DEBUG=true
      - DEBUG_SUSPEND_FLAG=n
   order-service:
-    image: debezium-examples/outbox-order-service:latest
+    image: debezium-example/outbox-order-service:${DEBEZIUM_VERSION}
```

Comment:
I tried to use `debezium-examples` consistently for all example images lately. So I'd prefer to stick to that (not all examples follow that pattern yet, some still use just `debezium`, which should get changed when touching such example).

Reply:
Fixed